app/helpers/mailer_helper.rb

```diff
@@ -16,6 +16,14 @@ def observer_text(observation, reason = nil)
     text + "."
   end
 
+  def property_display_value(field)
```

Comment:
I added a very similar method in https://github.com/18F/C2/pull/1079/files as part of the proposal attributes refactor so you may want to remove so we don't have merge conflicts.